tree-sitter-stack-graphs/src/cli/match.rs

```diff
@@ -0,0 +1,161 @@
+// -*- coding: utf-8 -*-
+// ------------------------------------------------------------------------------------------------
+// Copyright © 2021, stack-graphs authors.
+// Licensed under either of Apache License, Version 2.0, or MIT license, at your option.
+// Please see the LICENSE-APACHE or LICENSE-MIT files in this distribution for license details.
+// ------------------------------------------------------------------------------------------------
+
+use anyhow::anyhow;
+use clap::Args;
+use clap::ValueHint;
+use colored::Colorize;
+use std::path::Path;
+use std::path::PathBuf;
+use tree_sitter::CaptureQuantifier;
+use tree_sitter::Node;
+
+use crate::cli::parse::print_node;
+use crate::loader::FileReader;
+use crate::loader::Loader;
+use crate::NoCancellation;
+
+use super::parse::parse;
+use super::util::ExistingPathBufValueParser;
```

Comment:
Not really, I think. Looking through the code, it seems quite arbitrary. Perhaps it would be best to settle on using `crate` only, since those are easiest to interpret?

Reply:
I've used `crate` here. Might change other places whenever touching that code.